components/release-mgmt-dashboard/tslint.json

```diff
@@ -69,7 +69,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
Hrm, I'd have expected either one of:
1. leaving this file alone (i.e. ignoring this component)
2. updating this component's tslint in the same manner as the other ones, and updating this file.